Source/Scene/ClippingPlane.js

```diff
@@ -146,9 +149,9 @@ define([
     }
 
     defineProperties(UpdateChangedCartesian3.prototype, {
-        x : makeGetterStter('x'),
-        y : makeGetterStter('y'),
-        z : makeGetterStter('z')
+        x : makeGetterSetter('x'),
+        y : makeGetterSetter('y'),
+        z : makeGetterSetter('z')
```

Comment:
I would actually just heed @pjcozzi's advice here and not do the separate `makeGetterSetter` that I had suggested, just in case. https://github.com/AnalyticalGraphicsInc/cesium/pull/6201#discussion_r173656781